apps/etheroscope_db/lib/etheroscope_db/application.ex

```diff
@@ -8,22 +8,7 @@ defmodule EtheroscopeDB.Application do
   def start(_type, _args) do
     children = []
 
-    create_database()
```

Comment:
Where did this end up going?

Reply:
See message here https://github.com/Etheroscope/backend-phoenix/pull/5/commits/460d5ee6cb89944b6759a2a848677bc4157fcfd5